client/src/app.js

```diff
@@ -1352,6 +1352,24 @@ async function launchDisco(arg, app) {
 	if (!arg.genome) throw '"genome" parameter missing'
 	const genomeObj = app.genomes[arg.genome]
 	if (!genomeObj) throw 'unknown genome'
-	const _ = await import('#plots/disco/launch.adhoc.ts')
-	return await _.launch(arg.disco, genomeObj, app.holder0)
+	if (arg.disco.sample_id) {
```

Comment:
thanks. please document `.disco{}` parameter in https://github.com/stjude/proteinpaint/wiki/Embedding